go.work

```diff
@@ -1,5 +1,7 @@
-go 1.19
+go 1.21
```

Comment:
I don't know how go works but in other places you're installing 22.4

Reply:
Yeah 1.21 is the minimum version required to use Go templates. I will update this to 1.24 for consistency, good point. Nothing should break.